scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
not much to discuss then :grin: 

also, converting it to opt-in is as simple as setting [`disable_predictable_nin`](https://github.com/debian-pi/raspbian-ua-netinst/pull/391/files#diff-ef22c8fabe7f21fa3c3ab45495d6774bR28) to 0